src/h5web/metadata-viewer/MetadataViewer.tsx

```diff
@@ -6,7 +6,6 @@ import EntityInfo from './EntityInfo';
 import styles from './MetadataViewer.module.css';
 
 interface Props {
-  key: string;
```

Comment:
Unused prop. Also, we no longer need a `key` on `MetadataViewer` since `selectedEntity` is no longer retrieved asynchronously; it is always in sync with `selectedLink`.